### PR TITLE
collatz-conjecture: add to track

### PR DESCRIPTION
**Pull request by sjwarner-bp**
Add to track as part of fleshing out track, in preparation for launch.

I've not put the logic in a function - I figured that this should be ok, but let me know what you think!

### CODE REVIEW

File: exercises/collatz-conjecture/collatz_test.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  [ "$status" -eq 0 ]
+  [ "$output" -eq 152 ]
+}
+
+@test "zero is an error" {
+  skip
+  run bash collatz.sh 0
+
+  [ "$status" -eq 1 ]
```
Comment (budmc29) [inline exercises/collatz-conjecture/collatz_test.sh]: This expects an error.

File: exercises/collatz-conjecture/collatz_test.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+@test "negative value is an error" {
+  skip
+  run bash collatz.sh -15
+
+  [ "$status" -eq 1 ]
```
Comment (budmc29) [inline exercises/collatz-conjecture/collatz_test.sh]: Expects error as well.

File: exercises/collatz-conjecture/collatz_test.sh (hunk 1 above)
Comment (sjwarner-bp) [reply exercises/collatz-conjecture/collatz_test.sh]: Would you agree the error should be tested by something like:

```
[ "$output" -eq "Error: invalid input" ]
```

and that the error should be echo'ed out? 🙂

**Comment by sjwarner-bp** [comment]
Hopefully this is more like what you were expecting 🙂

**Comment by budmc29** [comment]
Thank you.

File: exercises/collatz-conjecture/collatz_test.sh (hunk 1 above)
Comment (budmc29) [reply exercises/collatz-conjecture/collatz_test.sh]: Yes.

**Pull request merged**